test/e2e/e2e.go

```diff
@@ -94,11 +92,11 @@ func setupHostRedirectPod(f *framework.Framework, node *v1.Node, exContainerName
 			Containers: []v1.Container{
 				{
 					Name:    tcpServer,
-					Image:   agnhostImage,
+					Image:   images.AgnHost(),
```

Comment:
The images stuff seems like it could be a separate commit

Reply:
done